engine/platform/sdl/vid_sdl.c

```diff
@@ -257,9 +257,13 @@ static void R_InitVideoModes( void )
 {
 	char buf[MAX_VA_STRING];
 #if SDL_VERSION_ATLEAST( 2, 0, 0 )
-	int displayIndex = 0; // TODO: handle multiple displays somehow
+	int displayIndex = SDL_GetWindowDisplayIndex(host.hWnd);
 	int i, modes;
 
+    if (displayIndex < 0) {	
```

Comment:
Please reformat your code.

Opening bracket must be on new line, and there should be spaces in parentheses.

There is provided Uncrustify config in the repository root for that.